### PR TITLE
Update CNN.com related ruleset

### PR DESCRIPTION
**Pull request by cschanaj**
- [x] cnn.com
 - [x] amp.cnn.com
 - [x] cnnpressroom.blogs.cnn.com
 - [x] i.cdn.cnn.com
 - [x] i2.cdn.cnn.com
 - [x] commercial.cnn.com
 - [x] edition.cnn.com
 - [x] go.cnn.com
 - [x] money.cnn.com
 - [x] buzz.money.cnn.com
 - [x] cgi.money.cnn.com
 - [x] jobsearch.money.cnn.com
 - [x] markets.money.cnn.com
 - [x] portfolio.money.cnn.com
 - [x] realestate.money.cnn.com
 - [x] www.money.cnn.com
 - [x] rss.cnn.com
 - [x] searchapp.cnn.com
 - [x] www.cnn.com

### CODE REVIEW

File: src/chrome/content/rules/CNN.com.xml [hunk 1]
```diff
@@ -1,68 +1,55 @@
...
-
-	Nonfunctional domains:
-
-		- cnn.com subdomains:
-
-			- ^ *	
-			- ads *
-			- edition **
-			- ireport *
-			- mexico ***
```
Comment (J0WI) [inline src/chrome/content/rules/CNN.com.xml]: This subdomain does still exist

File: src/chrome/content/rules/CNN.com.xml [hunk 2]
```diff
@@ -1,68 +1,55 @@
...
-			- (www.)?money *
-			- realestate.money *
```
Comment (J0WI) [inline src/chrome/content/rules/CNN.com.xml]: This subdomain does still exist

File: src/chrome/content/rules/CNN.com.xml [hunk 3]
```diff
@@ -1,68 +1,55 @@
...
-			- sportsillustrated **
-			- svcs *
-			- transcripts **
-			- us **
```
Comment (J0WI) [inline src/chrome/content/rules/CNN.com.xml]: This subdomain does still exist

**Comment by J0WI** [comment]
See also cnn.net

**Comment by cschanaj** [comment]
- [ ] debates.cnn.com
- [ ] lending.cnn.com
- [ ] podcast.cnn.com
- [ ] religion.cnn.com
- [ ] smartlink.cnn.com
- [ ] tours.cnn.com
- [ ] travel.cnn.com
- [ ] weather.cnn.com
- [ ] finance.fortune.cnn.com
- [ ] management.fortune.cnn.com
- [ ] tech.fortune.cnn.com
- [ ] cnn.net

**Pull request merged**